### ISSUE TITLE
Property 'match' does not exist for commands

### ISSUE THREAD

**Issue by rooton** [telegraf/telegraf#1938]
* Telegraf.js Version: 4.15.3
* Node.js Version: 20.10.0

## Minimal Example Code Reproducing the Issue

```ts
bot.command(/cmd_(\\d+)/), (async (ctx) => {
    const data = ctx.match[1] // <= Property 'match' does not exist
    ....
})
```

## Expected Behavior

Property 'match' exist.

It is works for hears and action. Support for regexp for command declared, but not fully implemented.

**Comment by MKRhere** [comment]
Thanks, will add to the upcoming release.

**Comment by MKRhere** [comment]
Published to npm as release candidate [4.16.0-rc.2](https://www.npmjs.com/package/telegraf/v/4.16.0-rc.2). Stable release when #1949 is merged.
